specs/core/0_beacon-chain.md

```diff
@@ -400,6 +404,19 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 }
 ```
 
+#### `WithdrawalData`
+
+```python
+{
+    # Withdrawal credentials
+    'withdrawal_credentials': 'hash32',
```

Comment:
Consider renaming to `credentials` (the context of a withdrawal is captured in the data structure name).